upf/upf.h

```diff
@@ -626,6 +630,9 @@ typedef struct
   /* DPO locks */
   u32 dpo_locks;
 
+  /* TEID table by choose_id */
+  u32 *teid_by_choose_id_table;
```

Comment:
what do think about renaming this to `teid_by_chid` ??

* `chid` is IMHO clear enough
* a `_table` suffix is not used in VPP or UPG for any of the lookup tables, the usual naming is just `x_by_y`

Also, what about changing all other instance of `choose_id` to `chid` ??